Gemfile

```diff
@@ -1,2 +1,5 @@
 source "https://www.rubygems.org"
 gemspec
+
+gem 'daru', :path => "/home/sameer/gitrepos/daru/"
```

Comment:
I know. I know. I'll remove that once the testing is done its production ready :P 

I need to do that because it frequently so happens that I need to make changes in daru to make something in statsample work.